### PR TITLE
feat(user_manual): document system address book

### LINKED ISSUE #10021: Document system addressbook
For https://github.com/nextcloud/server/issues/19575

## Notes

* If someone puts all profile properties to *private*, there is no system contact. Only users with at least one publishable property are added to the system address book.

### PR DESCRIPTION
**Pull request by miaulalala**
### ☑️ Resolves

* Fix #10021

### 🖼️ Screenshots

| Before | After |
|--------|--------|
| screencapture-127-0-0-1-8000-userpreferences-html-2023-05-04-16_51_34 | screencapture-127-0-0-1-8000-userpreferences-html-2023-05-04-16_47_45 |
| screencapture-127-0-0-1-8000-groupware-contacts-html-2023-05-04-16_50_48 | screencapture-127-0-0-1-8000-groupware-contacts-html-2023-05-04-16_48_24 |

### CODE REVIEW

File: user_manual/userpreferences.rst [hunk 1]
```diff
@@ -47,3 +43,28 @@ include the following:
 .. note:: Available options and settings depending on your administrator's configuration.
    If you are not able to change the password or the display name in your personal settings,
    please contact your administrator for help.
+
+
+Sharing your data in the global address book
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Some administrators decide to share their global address book with other Nextcloud instances (so called *Trusted Servers*) or even with the wider world. 
+This is helpful when two instances want to work closely together, or when people want to use Nextcloud as a virtual telephone book for others to peruse.
```
Comment (ChristophWurst) [inline user_manual/userpreferences.rst:52]: peruse? :monocle_face:

Comment (miaulalala) [reply user_manual/userpreferences.rst:52]: browse?

**Comment by ChristophWurst** [comment]
There is no documentation that wouldn't be true with the current SAB implementation. We can already merge this PR, even if https://github.com/nextcloud/server/pull/37734 is not in yet.

**Comment by ChristophWurst** [comment]
Conventional commits are not checked here but fyi `feat(user_manual)` is an invalid scope. I know because I made the mistake myself until @st3iny pointed it out. The scope should only be alphanumeric, so `feat(usermanual)`

:ghost:

**Review by ChristophWurst — APPROVED** [review]

**Comment by ChristophWurst** [comment]
we have not branched off yet, master=27

**Pull request merged**